### PR TITLE
Batch insertion of payments

### PR DESCRIPTION
**Pull request by DelusionalLogic**
How about this?

image

Testing this is pretty difficult (it's mostly js), so we should probably do some smoke testing.

### CODE REVIEW

**Comment by VirtualSatai** [comment]
It should probably be clear that the field on the right is in hundreds of a stregdollar.

**Comment by DelusionalLogic** [comment]
It's in stregøre actually

**Comment by hrjakobsen** [comment]
With the latest commit it is no longer possible to reach the delete button by pressing tab. This is due to a request from the treasurer.

File: stregsystem/templates/admin/stregsystem/batch_payment_done.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% extends "admin/base_site.html" %}
+
+{% block title %}Brødrazzia{% endblock %}
```
Comment (hrjakobsen) [inline stregsystem/templates/admin/stregsystem/batch_payment_done.html]: This is not the Brødrazzia page

File: stregsystem/templates/admin/stregsystem/batch_payment.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{% extends "admin/base_site.html" %}
+{% load static %}
+
+{% block title %}Brødrazzia{% endblock %}
```
Comment (hrjakobsen) [inline stregsystem/templates/admin/stregsystem/batch_payment.html]: This is not Brødrazzia

File: stregsystem/views.py [hunk 3]
```diff
@@ -210,3 +217,28 @@ def menu_sale(request, room_id, member_id, product_id=None):
     # Refresh member, to get new amount
     member = Member.objects.get(pk=member_id, active=True)
     return usermenu(request, room, member, product, from_sale=True)
+
+
+@staff_member_required()
+def batch_payment(request):
+    PaymentFormSet = forms.modelformset_factory(
+        Payment,
+        fields=("member", "amount"),
```
Comment (hrjakobsen) [inline stregsystem/views.py]: This allows for negative amounts, which is not intended (right?)

File: stregsystem/templates/admin/stregsystem/batch_payment_done.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+{% block breadcrumbs %}<div class="breadcrumbs"><a href="../../../">Hjem</a>&nbsp;&rsaquo;&nbsp;<a href="../../">Stregsystem</a>&nbsp;&rsaquo;&nbsp;<a>Razzia</a>&nbsp;&rsaquo;&nbsp;Bread</div>{% endblock %}
```
Comment (hrjakobsen) [inline stregsystem/templates/admin/stregsystem/batch_payment_done.html]: Wrong breadcrumbs as well

File: stregsystem/templates/admin/stregsystem/batch_payment.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+{% block breadcrumbs %}<div class="breadcrumbs"><a href="../../../">Hjem</a>&nbsp;&rsaquo;&nbsp;<a href="../../">Stregsystem</a>&nbsp;&rsaquo;&nbsp;<a>Razzia</a>&nbsp;&rsaquo;&nbsp;Bread</div>{% endblock %}
```
Comment (hrjakobsen) [inline stregsystem/templates/admin/stregsystem/batch_payment.html]: Wrong breadcrumbs

**Comment by hrjakobsen** [comment]
When multiple payments for the same user are submitted, both payments are saved but the user has a wrong balance.

**Comment by DelusionalLogic** [comment]
I think it's because of the way transactions are handled. I completely forgot that I have to actually add the amount to the users balance. Woops

File: stregsystem/views.py (hunk 3 above)
Comment (DelusionalLogic) [reply stregsystem/views.py]: It is

**Comment by DelusionalLogic** [comment]
It should be fixed now. Please test!

**Pull request merged**